app/controllers/items_controller.rb

```diff
@@ -50,16 +50,15 @@ def item_params
   def set_show_headline(item, review_count)
     headline = "Reviews"
     if review_count > 0
-      rating_sum = item.reviews.pluck(:rating).inject { |sum, n| sum + n }
-      rating_count = item.reviews.count
-      average_review = (rating_sum.to_f / rating_count).round(2)
-
-      headline += " - Average Review Score: #{average_review}"
+      headline += " - Average Review Score: #{calculate_average_review(item)}"
```

Comment:
See comments below - this method should be moved out of the controller, and it should be called like `item.average_review_rating`